src/Packagist/Api/Result/Package.php

```diff
@@ -50,6 +50,11 @@ class Package extends AbstractResult
     protected $favers;
 
     /**
+     * @var boolean
```

Comment:
`bool`, same on other places. You can run PHP-CS-Fixer. See `phpdoc_scalar` on https://github.com/FriendsOfPHP/PHP-CS-Fixer.